caproto/server/records.py

```diff
@@ -3641,7 +3473,7 @@ class HistogramFields(RecordFieldGroup):
 
 
 @register_record
-class LonginFields(RecordFieldGroup):
+class LonginFields(RecordFieldGroup, _Limits):
```

Comment:
I have not checked the other changes to ensure that switching in `_Limits` doesn't affect anything.

However, it's obvious here that it's incorrect: `longin` limits should _not_ be `ChannelType.DOUBLE`, they should be `ChannelType.LONG`.